perun-wui-profile/src/main/java/cz/metacentrum/perun/wui/profile/pages/settings/altPasswords/AltPasswordsView.ui.xml

```diff
@@ -0,0 +1,67 @@
+<ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
+             xmlns:b='urn:import:org.gwtbootstrap3.client.ui'
+             xmlns:b.html='urn:import:org.gwtbootstrap3.client.ui.html'
+             xmlns:p='urn:import:cz.metacentrum.perun.wui.widgets'
+             xmlns:g='urn:import:com.google.gwt.user.client.ui'
+             xmlns:b.gwt="urn:import:org.gwtbootstrap3.client.ui.gwt"
+>
+
+	<ui:style>
+	</ui:style>
+
+	<!-- Import CSS / GSS -->
+	<ui:with field="res" type="cz.metacentrum.perun.wui.profile.client.resources.PerunProfileResources"/>
+
+    <!-- Import Translation -->
+    <ui:with field='trans' type='cz.metacentrum.perun.wui.profile.client.resources.PerunProfileTranslation'/>
+
+    <!-- Import place tokens -->
+    <ui:with field="placeTokens" type="cz.metacentrum.perun.wui.profile.client.resources.PerunProfilePlaceTokens"/>
+
+    <b.html:Div addStyleNames="{res.gss.page}">
+        
+        <b:Heading size="H2" marginTop="5" >
+
+            <b:Breadcrumbs addStyleNames="{res.gss.perunBreadcrumb}">
+                <b:AnchorListItem icon="GEAR" text="{trans.menuSettings}"
+                                  targetHistoryToken="{placeTokens.getSettings}"/>
+                <b:ListItem text="{trans.menuAltPasswords}" />
+            </b:Breadcrumbs>
+        </b:Heading>
+
+        <b.html:Div>
+            <p:PerunButton ui:field="backButton" text="{trans.back}" icon="CHEVRON_LEFT" type="DEFAULT" marginBottom="10" marginTop="10"/>
+        </b.html:Div>
+
+        <b:Heading size="H4" text="{trans.generatePassword}"/>
+        <b:TextBox ui:field="passwordBox" placeholder="{trans.passwordDescription}"/>
+        <p:PerunButton ui:field="addAltPasswordButton" type="SUCCESS" text="{trans.addAltPassword}" icon="PLUS" marginTop="10" marginBottom="10"/>
```

Comment:
I would prefer textbox and button on single line. Try wrapping it in ``PerunToolbar`` if I recall the widget correctly. Also we should be consistent in language. So if heading says generate alternative password, then button shouldn't be "add new password", but more like simple "Generate".

Reply:
i couldn't mangage to get PerunToolbar to work so I used Hozrizonatal Panel insetead